src/raw_bmp.rs

```diff
@@ -74,6 +81,52 @@ impl<'a> RawBmp<'a> {
     pub fn pixels(&self) -> RawPixels<'_> {
         RawPixels::new(self)
     }
+
+    /// Returns the raw color of a pixel.
+    ///
+    /// Returns `None` if `p` is outside the image bounding box. Note that this function doesn't
+    /// apply a color map, if the image contains one.
+    pub fn pixel(&self, p: Point) -> Option<u32> {
+        let width = self.header.image_size.width as i32;
+        let height = self.header.image_size.height as i32;
+
+        if p.x < 0 || p.x >= width || p.y < 0 || p.y >= height {
+            return None;
+        }
+
+        let mut row_chunks = self.image_data.chunks_exact(self.header.bytes_per_row());
+        let row = match self.header.row_order {
+            RowOrder::BottomUp => row_chunks.nth_back(p.y as usize),
+            RowOrder::TopDown => row_chunks.nth(p.y as usize),
+        }?;
+
+        match self.header.bpp {
+            Bpp::Bits1 => RawDataSlice::<RawU1, LittleEndian>::new(row)
+                .into_iter()
+                .nth(p.x as usize)
```

Comment:
Using nth() seems to be rather expensive for random pixel lookup.
IIn Bmp format i should be possible to lookup pixels in the raw data stream in O(1) time.

Reply:
There is a specialized implementation of `nth` which is O(1).

Reply:
Where?

Reply:
`RawDataSlice` and the iterators are included in `embedded-graphics`: https://github.com/embedded-graphics/embedded-graphics/blob/master/src/iterator/raw.rs

Reply:
Cool. Does that also count for the nth in line 100?

Reply:
Yes, `Chunks` does also implements `nth`: https://doc.rust-lang.org/src/core/slice/iter.rs.html#1501

There might be some performance gain by not using `Chunks`, but that could be explored later. My focus for this PR was getting a working implementation not necessarily the fastest.

Reply:
Very nice. Looks good to me then.